src/modules/esl-popup/core/esl-popup.ts

```diff
@@ -163,6 +158,7 @@ export class ESLPopup extends ESLToggleable {
    * Removes CSS classes and update a11y by default.
```

Comment:
Comment needs small correction: "Removes CSS classes and updates a11y by default."